.nuget/NuGet.Config

```
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear />
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
```

small nit: update to v3 feed while editing this file:
```suggestion
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```